src/reference/asciidoc/kafka.adoc

```diff
@@ -326,3 +326,23 @@ public void listen(@Payload String foo,
     ...
 }
 ----
+
+===== Handling Duplicates
+
+In certain scenarios, such as rebalancing, a message may be redelivered that has already been processed.
+The framework cannot know whether or not such a message has been processed or not, that is an application-level
+function.
+This is known as the http://www.enterpriseintegrationpatterns.com/patterns/messaging/IdempotentReceiver.html[Idempotent
+Receiver] pattern and Spring Integration provides an http://docs.spring.io/spring-integration/reference/html/messaging-endpoints-chapter.html#idempotent-receiver
+[implementation thereof].
+
+The Spring for Apache Kafka project also provides some assistance by means of the `DeduplicatingMessageListenerAdapter`
+and `DeduplicatingAcknowledgingMessageListenerAdapter` classes, which can wrap your `MessageListener` or
```

Comment:
If you agree with me about camel case, these two mentionings should be addresses as well.